### ISSUE TITLE
After convert a mobilenet_v1 from pb model to lite, the model accuracy reduction for 12%

### ISSUE THREAD

**Issue by Changkou** [tensorflow/tensorflow#22166]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:None
- **TensorFlow installed from (source or binary)**:binary
- **TensorFlow version (use command below)**:1.9.0
- **Python version**:2.7
- **Bazel version (if compiling from source)**:1.6.1
- **GCC/Compiler version (if compiling from source)**:5.4.0
- **CUDA/cuDNN version**:9.0/7.1
- **GPU model and memory**:8GB
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.
I converted the pb-formatted mobilenet_v1 to lite with the following code:
python tflite_convert.py \
--output_file=/data2/LIBY0711/trained_model/quantize_0827/mobilenet_v1_quant_lr_0.005_GD_0.5_decay_int8/pb_model2/sku_classify_scale-66.tflite \
--graph_def_file=/data2/LIBY0711/trained_model/quantize_0827/mobilenet_v1_quant_lr_0.005_GD_0.5_decay_int8/pb_model2/frozen_inference_graph_skip.pb \
--inference_type=QUANTIZED_UINT8 \
--input_shape=1,224,224,3 \
--input_arrays=input \
--default_ranges_min=0 \
--default_ranges_max=6 \
--output_arrays=MobilenetV1/Predictions/Softmax \
--mean_values=0 \
--std_dev_values=1

However, the precision of lite after transformation is 12% lower than that of pb. I don't know if I'm setting the parameters incorrectly what else is going to be the problem, hopefully that's answered

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by gargn** [comment]
As documented [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/toco/g3doc/cmdline_reference.md) the `default_ranges_min` and `default_ranges_max` can lead to a drop in accuracy:

> --default_ranges_min, --default_ranges_max. Type: floating-point. Default value for the (min, max) range values used for all arrays without a specified range. Allows user to proceed with quantization of non-quantized or incorrectly-quantized input files. These flags produce models with low accuracy. They are intended for easy experimentation with quantization via "dummy quantization".

How are you generating the model? Additionally, can you provide the model file to help with debug what the conversion command should be?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tofulawrence** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
